### PR TITLE
chore: Remove unused dependencies

### PR DESCRIPTION
**Pull request by gkarat**
No jira.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,8 +115,7 @@
     "stylelint-scss": "^3.21.0",
     "ts-patch": "^3.1.2",
     "typescript": "^5.4.5",
-    "webpack-bundle-analyzer": "^4.5.0",
-    "whatwg-fetch": "^3.6.2"
+    "webpack-bundle-analyzer": "^4.5.0"
```
Comment (bastilian) [inline package.json:118]: I think we can also remove the analyser stuff. If we configure this we should do it somewhere central to include in all apps.

Comment (gkarat) [reply package.json]: Yeah, forgot about this one

**Comment by bastilian** [comment]
Do we use the "html-webpack-plugin" package? I think that may be remnant of some previous webpack config.

**Review by bastilian — APPROVED** [review]
LGTM! Everything appears to still work as expected! Thank you @gkarat!

**Pull request merged**